docs/proposals/volumes.md

```diff
@@ -362,7 +379,7 @@ The Kubelet should be modified to perform ownership and label management when re
 For ownership management the criteria are:
 
 1.  The `pod.Spec.SecurityContext.FSGroup` field is populated
-2.  The volume builder returns `true` from `SupportsOwnershipManagement`
+2.  The volume builder `GetAttributes` returns `true` for `SupportsOwnershipManagement`
 
 Logic should be added to the `mountExternalVolumes` method that runs a local `chgrp` and `chmod` if
```

Comment:
I dislike adding detailed code to a proposal, especially adding it later, because it's completely unclear what is present and what is future.  This code will ALWAYS be out of sync with real code.  It should just be pseudocode